docs/specs/shapes.md

```diff
@@ -7,6 +7,51 @@ The graphical elements are divided in 4 categories:
 * [Styles](#shape-style), that define the visual appearance of shapes
 * [Modifiers](#modifier) alter the curves of the shapes
 
+## Shape Rendering Model
```

Comment:
Should we mention opacity in a group? Specifically it should be applied to the whole group rather than individual shapes within it. (Although I know at least rlottie doesn't do that properly)

Reply:
Good idea, done.